auth-web/src/assets/scss/theme.scss

```diff
@@ -68,6 +68,7 @@ $app-blue: #1669bb; // same as the Vuetify theme primary
 $app-lt-blue: #E2E8EE; // LOW OPACITY PRIMARY BLUE
 $app-background-blue: #E4EDF7; // Hover Background Blue
 $app-dk-blue: #38598a;
+$app-green: #2E8540;
```

Comment:
For Added label, ie.

![image](https://github.com/bcgov/sbc-auth/assets/10002889/fbe76ac6-17f3-4d62-b026-79527440296e)